CHANGELOG.md

```diff
@@ -18,6 +22,9 @@
 - Rename component.TraceReceiver to component.TracesReceiver #1975
 - Rename component.TraceProcessor to component.TracesProcessor #1976
 - Rename component.TraceExporter to component.TracesExporter #1975
+
+## 💡 Enhancements 💡
```

Comment:
Why did you add this? The enhancements section is below

Reply:
oh this is a typo from the last merge from master, fixing now